_notes/why_must_i_learn_this.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Why Must I Learn This?
+---
+
+# “Why” matters
+
+“Why must I learn this?"
+
+I always ask this question as a kid. 
+
+The common replies are:
+
+- "I don't really know"
+- “Because it’s tested in the exam"
+- "Just do it, you will understand why next time"
+- “You will use it next year if you take X subject”
+
+Honestly, these answers never really satisfied my curiosity.
+
+Soon I stop asking why.
+
+Eventually I just followed mindlessly along.
+
+Though, that nagging feeling that something was off always stayed with me.
+
+Something about school just doesn’t quite resonate with real life.
+
+Now that I’m out of the system, I understand what’s missing.
+
+The difference between learning in school vs learning in real life.
+
+In school, we are often taught some piece of knowledge just-in-case (JIC) we need it.
+
+- We learn the structures of a benzene ring just-in-case (JIC) we need it.
+- We learn to get the second derivative of ln(x) just-in-case (JIC) we need it.
+- We learn Shakespeare’s poems just-in-case (JIC) we need it.
+
+“Just in case we need it” alone is a poor reason for learning.
+
+That’s why I’ve forgotten most of these things!
+
+My excuse is I have no reason to keep these knowledge in my brain.
+
+It’s poor because most people stop learning after they exit the system!
+
+In real life, we have no time to learn things just-in-case.
+
+We need better reasons to learn something new!
+
+A better system to keep us going.
```

Comment:
I dont think we should orient it as a system but rather a way

We find better ways for when we learn or how we learn